eclair-core/src/main/resources/reference.conf

```diff
@@ -191,63 +191,87 @@ eclair {
-      max-cltv = 1008              // max acceptable cltv expiry for the payment (1008 ~ 1 week)
-      fee-threshold-sat = 21       // if fee is below this value we skip the max-fee-pct check
-      max-fee-pct = 0.03           // route will be discarded if fee is above this value (in percentage relative to the total payment amount); doesn't apply if fee < fee-threshold-sat
-
-      // channel 'weight' is computed with the following formula: (channelFee + hop-cost) * (ratio-base + cltvDelta * ratio-cltv + channelAge * ratio-channel-age + channelCapacity * ratio-channel-capacity)
-      // the following parameters can be used to ask the router to use heuristics to find i.e: 'cltv-optimized' routes, **the sum of the four ratios must be 1**
-      ratio-base = 0.0
-      ratio-cltv = 0.05             // when computing the weight for a channel, consider its CLTV delta in this proportion
-      ratio-channel-age = 0.4       // when computing the weight for a channel, consider its AGE in this proportion
-      ratio-channel-capacity = 0.55 // when computing the weight for a channel, consider its CAPACITY in this proportion
-
-      // Virtual fee for additional hops
-      // Corresponds to how much you are willing to pay to get one less hop in the payment path
-      hop-cost-base-msat = 500
-      hop-cost-millionths = 200
-
-      // Examples:
-      // To optimize for fees only:
-      // ratio-base = 1
-      // ratio-cltv = 0
-      // ratio-channel-age = 0
-      // ratio-channel-capacity = 0
-      // hop-cost-base-msat = 0
-      // hop-cost-millionths = 0
-      //
-      // To optimize for shorter paths:
-      // ratio-base = 1
-      // ratio-cltv = 0
-      // ratio-channel-age = 0
-      // ratio-channel-capacity = 0
-      // hop-cost-base-msat = 10000 // High hop cost penalizes strongly longer paths
-      // hop-cost-millionths = 10000
-      //
-      // To optimize for successful payments:
-      // ratio-base = 0
-      // ratio-cltv = 0
-      // ratio-channel-age = 0.5 // Old channels should have less risk of failures
-      // ratio-channel-capacity = 0.5 // High capacity channels are more likely to have enough liquidity to relay our payment
-      // hop-cost-base-msat = 1000 // Less hops means less chances of failures
-      // hop-cost-millionths = 1000
-      //
-      // To optimize for fast payments:
-      // ratio-base = 0.2
-      // ratio-cltv = 0.5 // In case of failure we want our funds back as fast as possible
-      // ratio-channel-age = 0.3 // Older channels are more likely to run smoothly
-      // ratio-channel-capacity = 0
-      // hop-cost-base-msat = 10000 // Shorter paths should be faster
-      // hop-cost-millionths = 10000
-
-      mpp {
-        min-amount-satoshis = 15000 // minimum amount sent via partial HTLCs
-        max-parts = 5 // maximum number of HTLCs sent per payment: increasing this value will impact performance
+      control {
```

Comment:
I think not defining a `default` makes things harder to understand. It's not obvious that the `control` is also the `default`. Also that means experiments _may_ override settings, but _must_ override the percentage.